src/app/migrations/password/password.1.angular.txt

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  template: `
+    <form clrForm>
+      <clr-password-container>
+        <input clrPassword placeholder="input" name="input" [(ngModel)]="input" />
```

Comment:
The core example includes `label` here it is missing - is there a reason for that ?

Reply:
Yes, in the `core` example there is attribute `labelLayout="hidden-label"`